transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/AddTransferMethodFragment.java

```diff
@@ -447,6 +458,16 @@ public void showInputErrors(@NonNull final List<HyperwalletError> errors) {
                             widget.getView(mDynamicContainer).requestFocus();
                             focusSet = true;
                         }
+                        HyperwalletInsight.getInstance().trackError(requireContext(),
```

Comment:
@peter-joseph is there another ticket/PR that's addressing tracking errors that are not field (in-line messages) related, i.e the cases where we display the pop-up dialog?